package.json

```diff
@@ -129,6 +129,7 @@
     "react-virtualized": "^9.22.3",
     "reflux": "0.4.1",
     "regenerator-runtime": "^0.13.3",
+    "rrweb": "^1.1.3",
     "rrweb-player": "^0.7.13",
```

Comment:
The `rrweb-player` package depends on `rrweb`: https://github.com/rrweb-io/rrweb/blob/master/packages/rrweb-player/package.json#L28 
The `*-player` package uses svelte to add play/pause buttons and the like around the base `rrweb` iframe.

I'm bringing in `rrweb` explicitly so we can wrap the iframe ourselves, and use buttons and icons from the sentry component library instead. Also we'll have fewer svelte and a better react-oriented way to interact with the rrweb class.